controllers/interface.go

```diff
@@ -1,7 +1,7 @@
 package controllers
 
 import (
-	v1alpha "github.com/keikoproj/instance-manager/api/v1alpha1"
+	v1alpha "github.com/keikoproj/instance-manager/api/instancemgr/v1alpha1"
```

Comment:
why change the directory structure?

Reply:
because we are using the open source code generator to generate the clientset/informer, and they are expecting the apis should have the following directory structure:
[api(s)/$group/$version](https://github.com/kubernetes/code-generator/blob/release-1.26/generate-groups.sh#L67-L69)

most of the open source k8s controllers are following this pattern